src/Tasks/Common/Resources/Strings.resx

```diff
@@ -952,5 +952,9 @@ You may need to build the project on another operating system or architecture, o
     <value>NETSDK1215: Targeting .NET Standard prior to 2.0 is no longer recommended. See {0} for more details.</value>
     <comment>{StrBegin="NETSDK1215: "}</comment>
   </data>
-  <!-- The latest message added is TargetFrameworkIsNotRecommended. Please update this value with each PR to catch parallel PRs both adding a new message -->
+  <data name="MicrosoftNetSdkCompilersToolsetNotFound" xml:space="preserve">
+    <value>NETSDK1216: Package Microsoft.Net.Sdk.Compilers.Toolset is not downloaded but it is needed because your MSBuild and SDK versions are mismatched. Ensure the package is available in your NuGet source feeds and then run NuGet package restore from Visual Studio or MSBuild.</value>
```

Comment:
Do we know the version available at this point? It would be more helpful to tell the user "you need version X of package Y" than just "you need package Y". 

```suggestion
    <value>NETSDK1216: Package Microsoft.Net.Sdk.Compilers.Toolset is not downloaded but it is needed because your MSBuild and SDK versions are mismatched. Ensure version {0} of the package is available in your NuGet source feeds and then run NuGet package restore from Visual Studio or MSBuild.</value>
```

Note that this will require updating the callsite with formatting arguments.